### ISSUE TITLE
Sound fade off 

### ISSUE THREAD

**Issue by maxzam71** [OpenShot/openshot-qt#2603]
Hello
I faded the audio volume at the end of the movie, but in the final movie the audio suddenly stops.
I think it's a bug, what do you say?

**System Details**
 - Operating System / Distro: [e.g. Windows 10, Linux Mint 17.1]
 - OpenShot Version [e.g. 2.4.3]

**Screenshots**

**Comment by lnain07** [comment]
I have the same problem.
operating system windows 10
OpenShot version 2.4.3

**Comment by JTKreates** [comment]
I've come across the same problem. Openshot worked fine for about a week, but then I opened up my project after letting it sit for a couple days and the fade no longer works on audio.

I have tried restarting the program, restarting the computer, reinstalling openshot and deleting the `.openshot_qt` folder. None of these have worked so it's left me very confused how this could happen.

I am running Openshot version 2.4.3 on Windows 10 Home.
